src/main/java/org/folio/rs/controller/PubSubEventController.java

```diff
@@ -19,12 +33,53 @@
 
   private final RetrievalQueueService retrievalQueueService;
 
+  private static final ObjectMapper MAPPER = new ObjectMapper();
+
   @Override
-  public ResponseEntity<String> pubSubHandlersMovedEventPost(@Valid MovedEvent movedEvent) {
-    if (Objects.nonNull(movedEvent)) {
-      retrievalQueueService.processMovedEventRequest(movedEvent);
+  public ResponseEntity<String> pubSubHandlersMovedEventPost(@Valid PubSubEvent pubSubEvent) {
+    if (Objects.nonNull(pubSubEvent)) {
+      EventRequest eventRequest = null;
+      var payload = pubSubEvent.getEventPayload();
+      var documentContext = JsonPath.parse(payload);
+
+      try {
+        if (isPagedRequestCreated(documentContext)) {
```

Comment:
Please, remove requestType field from eventRequest.json .We dont need to populate that field anymore since we check request type via documentContext.read